analytics/analytics/tests/test_build_timings.py

```diff
@@ -21,7 +21,7 @@ def build_json_string():
 
 @pytest.mark.django_db
 def test_upload_build_timings(build_json_string):
-    upload_build_timings(build_json_string)
+    process_job(build_json_string)
```

Comment:
Out of curiosity, does this exercise a lot of your new code?

Reply:
This change was mainly done to keep the existing behavior consistent, although `process_job` does pretty much exercise all of my new code.